### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Link a Cluster-Link a Cluster_HD Insight_Color Contrast: Color contrast fails for Place holder text which is < 4.5:1 ratio

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/azure-tools-for-java#3626]
[Check out Accessibility Insights!](https://github.com/microsoft/azure-tools-for-java/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Colorcontrast; #MAS1.4.3

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)
MAS Violated: MAS1.4.3

**Note: Similar issue is observed in** 
1.Link a Cluster-Link a Cluster_Livy Service
2.Link SQL Server big data cluster_Link SQL Server big data cluster
3.Add Source Code-Create new scala class

**Repro Steps:**
1)Launch IntelliJ IDEA (Darcula and IntelliJ theme)
2)Tab till Create New Project button and press enter
3)Tab till view-->Tool Window-->Azure Explorer and press enter
4)Tab till signin-->copy and open & sign in to azure
5)Tab till select button and press enter
6)Tab till HDInsight and open context menu by using keyboard key
7)Tab till link a cluster and press enter
8)Verify whether Color contrast for Place holder text is >= 4.5:1 ratio

**Actual Result:**
Color contrast fails for Place holder text which is < 4.5:1 ratio

**Expected Result:**
Color contrast for Place holder text should be >= 4.5:1 ratio

**User Impact:**
Low vision user will find difficulty if the color contrast ratio is < 4.5:1

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![Bug_color contrast fails for placeholder text](https://user-images.githubusercontent.com/51435266/65573165-c0383980-df87-11e9-8df1-066b788c4006.png)

![Color contrast fails for placeholder text](https://user-images.githubusercontent.com/51435266/65679994-c4418580-e073-11e9-988e-3962b7bfe35a.png)

**Comment by t-rufang** [comment]
1. Contrast ratio issue in the `Link A Cluster` dialog will be fixed.
![Bug_color contrast fails for placeholder text](https://user-images.githubusercontent.com/51435266/65573165-c0383980-df87-11e9-8df1-066b788c4006.png)

2. For the `Create New Scala` Class dialog, it's supported by IntelliJ, we won't fix that one.
![Color contrast fails for placeholder text](https://user-images.githubusercontent.com/51435266/65679994-c4418580-e073-11e9-988e-3962b7bfe35a.png)

**Comment by v-lasiva** [comment]
@Alexandria
For the above mentioned 2 issues there is no contrast issue only in "high contrast" mode,
But in both "Darcula, Default" modes, there is a contrast issue for the place holder text.
As the bug is tagged with "Inavlid" tag please check the issue once.
Added attachments for reference.

![Bug_Normal mode](https://user-images.githubusercontent.com/55690326/69030471-b538dd00-09fd-11ea-95ef-babe054e3a72.png)
![Bug_High contrast mode](https://user-images.githubusercontent.com/55690326/69030472-b5d17380-09fd-11ea-911c-efbaf27e1f01.png)

**Comment by v-lasiva** [comment]
After discussing with SME based on their suggestions reactivating the bug.
The issue needs to meet contrast minimums in at least the "Default" theme.
Added attachment for reference.

![Bug_Normal mode](https://user-images.githubusercontent.com/55690326/69123835-5c824680-0ac8-11ea-9fde-2d2e1c2c57d2.png)

**Comment by t-rufang** [comment]
The textfield with hint is extended from class `com.intellij.ui.components.JBTextField`. Therefore this issue should be fixed by IntelliJ and it's out of the scope of our toolkit.

**Comment by Alexandria-R** [comment]
This can be closed as external to IntelliJ, then. @Amulya-95

**Comment by t-rufang** [comment]
- Importance of experience: Minimal. The hint text is hard for low vision user to see.
- Quality of experience: Managable. User can change to High Contrast theme to avoid color contrast too low issue.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by v-lasiva** [comment]
After discussing with SME, As they have approved closing this issue accordingly.

**Comment by v-lasiva** [comment]
After discussing with SME, As they have approved closing the issue accordingly.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749724
